libbeat/docs/outputconfig.asciidoc

```diff
@@ -106,10 +106,6 @@ The number of workers per configured host publishing events to Elasticsearch. Th
 is best used with load balancing mode enabled. Example: If you have 2 hosts and
 3 workers, in total 6 workers are started (3 for each host).
 
-===== port
```

Comment:
More as a historical remark: port was removed as it would not work will with `hosts`.